build-image/openhabian.pi-raspbian.conf

```diff
@@ -28,12 +28,13 @@ wifi_country=""
 # repo to clone from (if not master for build and test)
 repositoryurl=https://github.com/openhab/openhabian.git
 clonebranch=master
+
 # debug verbosity mode
-# unattended, unattended_debug (verbose output in log) or debug_maximum (show every command)
+# values: "unattended", "unattended_debug" (verbose output in log) or "debug_maximum" (show every command in either mode)
 mode=unattended
 
 # Java architecture mode
-# Use 32-bit or 64-bit
-java_arch=32-bit
+# values: "Zulu32", "Zulu64" or "AdoptOpenJDK"
+java_arch=64-bit
```

Comment:
Why switch to 64-bit default on RPi?

Reply:
Ouch, good catch. That was a copy 'n paste mistake.